test/jest-puppeteer-vuetify/src/vuetify-xpaths.ts

```diff
@@ -0,0 +1,174 @@
+type XPath = string;
+type Predicate = string;
+type Strings = string | Strings[] | undefined;
```

Comment:
```suggestion
type Strings = string | Strings[];
```

With `compilerOptions.strictNullChecks` (which you've enabled via `compilerOptions.strict`), use of the `?.` (which you seem to be using in all of the `Strings`-typed parameters) will automatically include `undefined`.

Reply:
I copied this from some online resource that mixed the two. If `?=` is the preferred standard syntax, I will use it.

Reply:
I turned on all the strictest things because I couldn't think of a reason not to, especially when starting a fresh project like this. In this case, I don't mind having both the `type Strings = ... | undefined` and the `?:`, because methods using the spread syntax apparently do not work with the `?:` syntax:
```
function classAsPredicate(...classes: Strings[]): Predicate {...}
```
Even if it's technically optional, I would rather have that `undefined` typed in to `Strings`.